api/config/credentials/test.yml.enc

```diff
@@ -0,0 +1 @@
+msZ338Es/pU9DHc/73+z3A45ziw+uHMpwQ7tVGVM1gBNl/mWjzqybuRGRnMEok1jq+oRy2bg8H03EmapMbaEwYeMVvjikMa2pUAFprEZs7J37j1GFcf7AaTOeNPAS5C1C1OIjeOOmx+i96KLfPt5xX2bxtPapc10OOvuJ5U57ZeM--fgIypbLI/J9YWvmS--qxDaiSNVHOadv2Qddv+Reg==
```

Comment:
I'm creating here some secrets for test environment. This was needed to be able to run rspec in test env, not dev env